configs/inpainting/AOT-GAN/AOT-GAN_512x512_places.py

```diff
@@ -0,0 +1,188 @@
+    dict(
+        type='Normalize',
+        keys=['gt_img'],
+        mean=[127.5] * 3,
+        std=[127.5] * 3,
+        to_rgb=False),
+    dict(type='GetMaskedImage'),
+    dict(
+        type='Collect',
+        keys=['gt_img', 'masked_img', 'mask'],
+        meta_keys=['gt_img_path']),
+    dict(type='ImageToTensor', keys=['gt_img', 'masked_img', 'mask'])
+]
+
+test_pipeline = [
+    dict(type='LoadImageFromFile', key='gt_img', channel_order='rgb'),
+    dict(
+        type='LoadMask',
+        mask_mode='set',
+        mask_config=dict(
+            mask_list_file=f'{mask_root}/val_places_mask_list.txt',
+            prefix=mask_root,
+            io_backend='disk',
+            flag='unchanged',
+            file_client_kwargs=dict())),
+    dict(
+        type='Crop',
+        keys=['gt_img'],
+        crop_size=(512, 512),
+        random_crop=False,
+    ),
+    dict(
+        type='Normalize',
+        keys=['gt_img'],
+        mean=[127.5] * 3,
+        std=[127.5] * 3,
+        to_rgb=True),
+    dict(type='GetMaskedImage'),
+    dict(
+        type='Collect',
+        keys=['gt_img', 'masked_img', 'mask'],
+        meta_keys=['gt_img_path']),
+    dict(type='ImageToTensor', keys=['gt_img', 'masked_img', 'mask'])
+]
+
+data_root = 'data/places365'
+
+data = dict(
+    workers_per_gpu=4,
+    train_dataloader=dict(samples_per_gpu=12, drop_last=True),
+    val_dataloader=dict(samples_per_gpu=1),
+    test_dataloader=dict(samples_per_gpu=1),
+    train=dict(
+        type=dataset_type,
+        ann_file=f'{data_root}/train_places_img_list.txt',
+        data_prefix=data_root,
+        pipeline=train_pipeline,
+        test_mode=False),
+    val=dict(
+        type=dataset_type,
+        ann_file=f'{data_root}/val_places_img_list.txt',
+        data_prefix=data_root,
+        pipeline=test_pipeline,
+        test_mode=True),
+    test=dict(
+        type=dataset_type,
+        ann_file=(f'{data_root}/val_places_img_list.txt'),
+        data_prefix=data_root,
+        pipeline=test_pipeline,
+        test_mode=True))
+
+optimizers = dict(
+    generator=dict(type='Adam', lr=0.0001, betas=(0.0, 0.9)),
+    disc=dict(type='Adam', lr=0.0001, betas=(0.0, 0.9)))
+
+lr_config = dict(policy='Fixed', by_epoch=False)
+
+checkpoint_config = dict(by_epoch=False, interval=10000)
+log_config = dict(
+    interval=100,
+    hooks=[
+        dict(type='TextLoggerHook', by_epoch=False),
+        dict(type='TensorboardLoggerHook'),
+        dict(type='PaviLoggerHook', init_kwargs=dict(project='mmedit'))
+    ])
+
+visual_config = dict(
+    type='VisualizationHook',
+    output_dir='visual',
+    interval=1000,
+    res_name_list=['gt_img', 'masked_img', 'fake_res', 'fake_img'],
+)
+
+evaluation = dict(interval=50000)
+
+total_iters = 500002
+dist_params = dict(backend='nccl')
+log_level = 'INFO'
+work_dir = './workdirs/aotgan_places'
+load_from = None
+resume_from = None
+workflow = [('train', 10000)]
+exp_name = 'AOT-GAN_512x512_places'
```

Comment:
We have a naming convention for the models. Would you mind use the convention below?
```
{model}_[model setting]_{backbone}_[refiner]_[norm setting]_[misc]_[gpu x batch_per_gpu]_{schedule}_{dataset}

```
An example can also be found [here](https://github.com/open-mmlab/mmediting/blob/master/configs/inpainting/partial_conv/pconv_256x256_stage1_8x1_celeba.py#L139)

Reply:
> Hi @quincylin1 , would you mind also providing the weights converted from the official weights?

The checkpoint can be named as follows:
```
{config_name}_{date}-{hash}.pth
```
For the hash, you can use `tools/publish_model.py` to generate.

Reply:
> We have a naming convention for the models. Would you mind use the convention below?
> 
> ```
> {model}_[model setting]_{backbone}_[refiner]_[norm setting]_[misc]_[gpu x batch_per_gpu]_{schedule}_{dataset}
> ```
> 
> An example can also be found [here](https://github.com/open-mmlab/mmediting/blob/master/configs/inpainting/partial_conv/pconv_256x256_stage1_8x1_celeba.py#L139)

<img width="709" alt="image" src="https://user-images.githubusercontent.com/33197366/161229535-d14adcd1-4547-4572-98bf-6cb39e50afc8.png">

@ckkelvinchan just wondering if this name is ok for this config: `AOT-GAN_512x512_4x12_places.py`

Reply:
Yeah I think it works.